test/fixtures/multiple_with_dns/main.tf

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+module "memstore" {
+  source = "../../../examples/multiple_with_dns"
```

Comment:
How are variables being passed through?

Reply:
They're hardcoded in `examples/multiple_with_dns/memorystore.tf`. You suggested I do that in a different module, so that's why I did that here.